contents/blog/data-management-feature.md

```diff
@@ -0,0 +1,96 @@
+
+![PostHog - write definition hover card](../images/blog/data-management-feature/write_definition_card.png)
+
+### Creating definitions in the new Data Management Tab
+
+Alternatively, click on view in the definition hover card and you'll be redirected to the new "Data Management" page. From here, explore a centralized place to manage all the event and event property definitions you have created.
+
+[WIP screenshot]
+
+### Creating definitions with the API
+
+You can also opt to create definitions with our API. Check out our documentation on how to use the [`event_definitions`](/docs/api/event-definitions) and [`property_definitions`](/docs/api/property-definitions) API endpoints.
+
+## How to use Data Management as a team (Best practices)
+
+1. **Rule of thumb**: The more definitions you define, the easier it will be for your teammates to understand what each event and property is describing. 
+
+Take a custom event sent as `sign_up` for example that is triggered on user sign up. Pretty self explanatory, right? Imagine you have a second app pushing data to the same PH instance sending sign up events as `sign up`. Or maybe you send a `sign up` event from your frontend and one from your backend. A new teammate using PostHog will find it impossible to differentiate between the two, potentially leading to incorrect conclusions and frustration. Describing these events properly can ensure your teammates find exactly what they're looking for.
+
+3. **Rule of thumb**: Official organization-wide events should be verified so that teammates can discover and use them easily. 
+
+Verified events will be prioritized in filters and other selection components to signal to the rest of the team that this event should be used in favor of other similar events.
+
+3. **Rule of thumb**: Tagging is an easy way to categorize events into custom groups, be it for a product, feature or new page. 
+
+We highly recommend using tags to organize events coming in from different parts of your product. 
+
+## We've moved some things around
+
+- The "Events & Actions" tab has been renamed to "Live Events".
+- "Actions", "Event Stats", and "Property Stats" have been moved into the new "Data Management" tab.
+- "Event Stats" has been renamed to "Events"
+- "Property Stats" has been renamed to "Event Properties"
+
+## Do I need to create event definitions before ingesting that event?
+
+Not at all! The awesome thing about definitions in PostHog is that they can be created, updated, or deleted at any time without affecting the event data that is already there. This means that you can create your definitions retroactively which nicely scales with your team's growing data needs.
+
+## What's next for Data Management?
+
+We're excited to get these data management features in, but we're just scratching the surface. We've prepared some documentation to help you get started with creating definitions and potential use-cases.
```

Comment:
We should expand a little more on the roadmap for data management here. Just a taster to give people a sense of our direction.